.github/workflows/tests.yml

```
@@ -0,0 +1,37 @@
+          cache: 'npm'
+
+      # Step 3: Install dependencies (if needed) and cache them
+      - name: Install Dependencies
+        run: npm install
+
+      # Step 4: Run the tests
+      - name: Run tests
+        env:
+          PORT: 8800
+          NODE_ENV: "test"
+        run: npm run test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)